Release/src/http/client/http_client_asio.cpp

```diff
@@ -82,7 +79,7 @@ using utility::conversions::details::to_string;
 using std::to_string;
 #endif
 
-#define CRLF std::string("\r\n")
+const std::string CRLF("\r\n");
```

Comment:
This should probably be static or moved into a details namespace to avoid conflicting symbols if the user static links.